### PR TITLE
added product categories

### PR DESCRIPTION
**Pull request by sattyframework**
Made a start on the product categories page. Used taxonomies for the categories. When you hover over the card the arrow button animates. 

Dekstop
All-of-our-products-Desktop

Mobile

### CODE REVIEW

File: resources/views/product-categories.antlers.html [hunk 1]
```diff
@@ -0,0 +1,14 @@
+<section class="productCategories bg-concrete">
```
Comment (Dunsters99) [inline resources/views/product-categories.antlers.html:1]: should this class name be `product-categories`?

File: resources/views/products/_categories.antlers.html [hunk 2]
```diff
@@ -0,0 +1,7 @@
+<div class="grid grid-cols-1 md:grid-cols-2 xl:grid-cols-3 gap-x-20 gap-y-60 pt-90 xl:pt-100">
```
Comment (Dunsters99) [inline resources/views/products/_categories.antlers.html:1]: double check that this is inside the grid using the Grid overlay helper

File: resources/views/products/_category-card.antlers.html [hunk 3]
```diff
@@ -0,0 +1,21 @@
+
+<div class="text-slate-dark">
```
Comment (Dunsters99) [inline resources/views/products/_category-card.antlers.html:2]: I'd add a class name for this element

File: resources/views/products/_category-card.antlers.html [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+
+  <div class="relative group">
+
+    <a class="" href="{{url}}">
+      {{ partial:helpers/media :media="media.0" class="w-full object-cover aspect-[4/3]" }}
```
Comment (Dunsters99) [inline resources/views/products/_category-card.antlers.html:7]: if check for the media

File: resources/views/products/_category-card.antlers.html [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+
+      <button class="btn-arrow btn-arrow--mustard absolute -bottom-30 right-20 z-20">
+        {{ svg src="arrow-right" class="group-hover:animate-[arrow-right-slide_1s_ease-in-out_infinite]" }}
```
Comment (Dunsters99) [inline resources/views/products/_category-card.antlers.html:10]: might be worth just adding this to the TW config so it doesn't clutter this file

File: resources/views/products/_categories.antlers.html (hunk 2 above)
Comment (sattyframework) [reply resources/views/products/_categories.antlers.html:1]: Yep all good
87646B73-733E-4F58-81F8-148C28E188FA

**Pull request merged**